app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -575,7 +594,6 @@ def vm_instance_defaults
         :memory_mb  => 768,
         :interfaces => [new_interface],
         :volumes    => [new_volume],
-        :scsi_controller => { :type => scsi_controller_default_type },
```

Comment:
I'm setting the default type in parse_args as its necessary to set the default controller type for all volumes and not only for one.